src/Illuminate/Collections/Arr.php

```diff
@@ -181,10 +181,13 @@ public static function exists($array, $key)
     /**
      * Return the first element in an array passing a given truth test.
      *
-     * @param  iterable  $array
-     * @param  callable|null  $callback
-     * @param  mixed  $default
-     * @return mixed
+     * @template TValue
+     * @template TFirstDefault
```

Comment:
I guess you can call this TDefault no?

Reply:
Was pretty much to replicate what was done for the other Collection classes:
https://github.com/search?q=repo%3Alaravel%2Fframework%20TFirstDefault&type=code